### PR TITLE
symbolz: skip un-symbolizable mappings

### PR DESCRIPTION
**Pull request by toddlipcon**
This fixes https://github.com/google/pprof/issues/339 by skipping
attempts to symbolize anything from system mappings like [vdso],
[vsyscall], etc.

### CODE REVIEW

File: internal/symbolz/symbolz.go [hunk 1]
```diff
@@ -45,6 +45,10 @@ func Symbolize(p *profile.Profile, force bool, sources plugin.MappingSources, sy
 			// Only check for HasFunctions as symbolz only populates function names.
 			continue
 		}
+		// Skip well-known system mappings
```
Comment (aalexand) [inline internal/symbolz/symbolz.go]: Doc for the function should be updated.

**Comment by aalexand** [comment]
You also have to sign the CLA.

**Comment by toddlipcon** [comment]
I've been through the CLA process a couple years ago and somehow it got lost. Unfortunately going through it again may take some days of clearance with my company's legal team etc. I was under the understanding that for "trivial bug fix" patches like this, CLAs aren't necessary?

**Comment by aalexand** [comment]
I don't know if there is an exemption from CLA based on the patch size, it's not in my domain of expertise so I have to merely follow the automated CLA pre-submit check and can't merge PRs with that check failed.

**Comment by aalexand** [comment]
@toddlipcon Do you plan to proceed with signing the CLA?  I guess alternatively I can just make the fix myself if you are fine with that.

**Comment by toddlipcon** [comment]
Sure, I'm fine with that. Sorry for radio silence, on PTO.

**Comment by aalexand** [comment]
Superseded by https://github.com/google/pprof/pull/368.

**Pull request closed without merging**